readthedocs/projects/models.py

```diff
@@ -1577,25 +1581,26 @@ def get_payload(self, version, build, event):
         ).isoformat()
 
         substitutions = {
-            '$event': event,
-            '$build.id': build.id,
-            '$build.commit': commit,
-            '$build.url': build_url,
-            '$build.docs_url': build_docsurl,
-            '$build.start_date': start_date,
-            '$organization.name': organization_name,
-            '$organization.slug': organization_slug,
-            '$project.slug': project.slug,
-            '$project.name': project.name,
-            '$project.url': project_url,
-            '$version.slug': version.slug,
-            '$version.name': version.verbose_name,
+            'event': event,
+            'build.id': build.id,
+            'build.commit': commit,
+            'build.url': build_url,
+            'build.docs_url': build_docsurl,
+            'build.start_date': start_date,
+            'organization.name': organization_name,
+            'organization.slug': organization_slug,
+            'project.slug': project.slug,
+            'project.name': project.name,
+            'project.url': project_url,
+            'version.slug': version.slug,
+            'version.name': version.verbose_name,
         }
         payload = self.payload
         # Small protection for DDoS.
         max_substitutions = 99
         for substitution, value in substitutions.items():
-            payload = payload.replace(substitution, str(value), max_substitutions)
+            payload = payload.replace(f'{{{{ {substitution} }}}}', str(value), max_substitutions)
+            payload = payload.replace(f'{{{{{substitution}}}}}', str(value), max_substitutions)
```

Comment:
I feel like this needs a comment. That's a whole like of `{`'s! Are they being escaped?

Reply:
yeah, that's `{{` to escape one `{`.